lib/storage.js

```diff
@@ -46,9 +47,7 @@ Store.prototype._createAppRef = function (id, fbRef, data) {
     this._removeAppRef(id);
   }
 
-  var process = this._processSnapshot;
-
-  function newBlock(blockId) {
+  var app = new App(this, id, fbRef, data, _firebase, function newBlock(blockId) {
```

Comment:
inlining this function is certainly ugly, but if I leave it as "defined in this function" as it is right now, jshint (rightly) compains it's not actually being used in the file that defined it.